README.md

```diff
@@ -1,5 +1,104 @@
-# Motorcycle.js DOM Driver 
+# Motorcycle.js DOM Driver [![Build Status](https://travis-ci.org/motorcyclejs/dom.svg?branch=develop)](https://travis-ci.org/motorcyclejs/dom)
 
-The Standard DOM Driver for motorcycle.js
+The Standard DOM Driver for Motorcycle. Built using [Snabbdom](https://github.com/paldepind/snabbdom) for it's modularity and it's faster *virtual-dom* implementation. This library exports the fantastic [hyperscript-helpers](https://github.com/ohanhi/hyperscript-helpers) library for ease-of-use.
 
-More information available soon!
+## Installing
+```
+$ npm install @motorcycle/dom
+```
+
+## Want to Contribute?
+
+If you found an issue or want to contribute code, please read
+the [contributing guidelines](https://github.com/motorcyclejs/motorcycle/blob/master/CONTRIBUTING.md).
+
+## Differences from [Cycle-DOM](https://github.com/cyclejs/cycle-dom)
+
+Many of the differences come from Snabbdom itself, and I highly recommend
+reading through their
+[documentation](https://github.com/paldepind/snabbdom#introduction)
+first to discover the differences between Matt Esch's *Virtual-DOM*.
+
+A less obvious difference is how this driver patches the DOM.
+
+For Example - Given the view `h1('Hello, World!')`  being patched to `<div id='#app'></div>` this is how it will look after being patched.
```

Comment:
> For Example - Given the view `h1('Hello, World!')`  being patched to `<div id='#app'></div>` this is how it will look after being patched.

For example, given the view `h1('Hello, World!')` being patched to `<div id='#app'></div>`, this is how it will look after being patched: